packages/design-tokens/docs/components/TokensTable.tsx

```diff
@@ -10,26 +12,36 @@ import {
 } from "../../../components/src/components/Table/";
 import { Token } from "../types/Token";
 import { buildTokensTableRows } from "../utils";
+import { TokenRow } from "../types/TokenRow";
 import { TOKEN_COLUMNS } from "./constants";
 import { TokenName } from "./TokenName";
 
 type TokenTypes = keyof typeof TOKEN_COLUMNS;
 
 export const TokensTable = ({
-  type,
   data,
 }: {
   type: TokenTypes;
```

Comment:
`type: TokenTypes;` can go away from the function signature since it's not used anymore!